docs/platform/getting-started/npm.md

```diff
@@ -1,42 +1,45 @@
 ---
 id: npm
-title: Adding your NPM auth token
-sidebar_label: Adding your NPM auth token
+title: 3. Access our packages
+sidebar_label: 3. Access our packages
 ---
 
+## Overview
+
 In order to be able to get access to the **Falcon Platform packages**, you will need to connect to our private npm registry.
 
-You can do this using an **authorisation token**. You can get this from your admin panel or by contacting us.
+You can do this using the **authorisation token** from [Step 2](/docs/platform/getting-started/dcloud#getting-your-npm-token). 
 
-To set up your token please follow these steps:
+You'll need to add your token to a [.npmrc file](https://docs.npmjs.com/cli/v7/configuring-npm/npmrc).
 
-1. Log in into <a href="https://npm.deity.io" target="_blank" rel="noreferrer noopener">https://npm.deity.io</a> (you can use any email) using credentials provided by DEITY:
+We recommend adding your token to a user config, it can however be added in your projects root.
 
-```javascript
-npm login --registry=https://npm.deity.io --scope=@deity
+## Setup
+
+### Add / Edit your .npmrc
+
+You'll need to either create or edit a `.npmrc` file. The user config can be found in your users root directory `~/.npmrc`.  If you don't have one, you can create one:
+
+```bash
+nano ~/.npmrc
```

Comment:
This seems to be assuming you're using unix (or linux?), is that sufficient?
Do we maybe also want to add instructions for other OSes? I feel like this is quite simple, so it may not be needed, but then we might still want to acknowledge that this may not work on an other OS.

Reply:
@TarVK - that's a good idea. I don't know the differences myself though. Do you?

Reply:
Not quite. On windows I would just use the file explorer and a text editor instead of terminal. Don't know of any convenient terminal based methods.

Reply:
I've added a note.